ui/cypress/e2e/a11y-basic.cy.js

```diff
@@ -0,0 +1,55 @@
+import { a11yLog } from '../support/custom';
+
+describe('Page a11y', () => {
+  describe('Homepage', () => {
+    it('passes a11y', () => {
+      cy.visit('/');
+      cy.get('[role="main"]');
+      cy.get('h1');
+      cy.injectAxe();
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
```

Comment:
```suggestion
      // eslint-disable-next-line cypress/i-think-youll-find-this-is-actually-necessary-waiting
```

/s

Reply:
it's so annoying! The solution is supposed to be 
a. perform a `cy.get({element which will be important in the context of the test})`
b. inject axe checker
c. check.

I found that _does_ work, but only about 7/10 times.